ksqldb-engine/src/main/java/io/confluent/ksql/engine/QueryMonitor.java

```diff
@@ -139,9 +139,7 @@ private void maybeRestartQueries() {
         final PersistentQueryMetadata queryMetadata = query.get();
         switch (queryMetadata.getState()) {
           case ERROR:
-            if (now >= retryEvent.nextRestartTimeMs()) {
```

Comment:
We can probably just remove this whole class, it does mark queries as healthy but I think that is just for use in this class

Reply:
removed

Reply:
With this class removed, it seems QueryMetadata.clearErrors would never be called (since we do not check if it has been resumed for long enough time anyways). Is the `queryErrors` queue inside QueryMetadata still needed?

Reply:
I plan on using that in the new uptime metric so even though it is not used right now it should be useful later